cterasdk/common/types.py

```diff
@@ -0,0 +1,96 @@
+from .object import Object
+
+
+class PolicyRule:
+
+    def __init__(self, value, criteria):
+        self.value = value
```

Comment:
I'm guessing you've named this attribute "value" to make it generic. But, I think it's too generic to understand what it is. Maybe something like "assignment" (which can be a name of plan, or an action or whatever).
Another way to address it is to have a sub-class with meaningful names and properties. For example:

```
class AutoAssignmentPolicyRule(PolicyRule):
  def __init__(self, plan, criteria):
    super().__init__(plan, criteria)
  @property
  def plan(self):
    return self.value
```

Reply:
Ok. I kept the converter. Changed "value" to "assignment" and added an option to pass the assignment value. Let me know what you think... if we adopt the subclass option than PolicyRule will need to inherit from `Object`

Reply:
OK